src/dom/webgl-image-sequence.ts

```diff
@@ -991,6 +990,12 @@ export class WebGlImageSequence {
         if (imageSource == this.lastDrawSource) {
             return;
         }
+
+        // Can't draw on a 0 area canvas
+        if (this.canvasWidth * this.canvasHeight === 0) {
```

Comment:
small detail but instead of multiplying the values (which can create a large number and would potentially need to create more memory for it), can you simply check each value instead? e.g.:

```
if (this.canvasWidth === 0 || this.canvasHeight === 0) { ... }
```

Reply:
Nice. Changed to` if (!this.canvasWidth || !this.canvasHeight) {`